src/realm/group.cpp

```diff
@@ -790,7 +790,9 @@ void Group::write(File& file, const char* encryption_key, uint_fast64_t version_
     REALM_ASSERT(file.get_size() == 0);
 
     file.set_encryption_key(encryption_key);
-    File::Streambuf streambuf(&file);
+    // Use big buffer to curb the cust of encryption
```

Comment:
cust/cost

Reply:
Oops

Reply:
I'll amend my commit and force push. I hope nobody is negatively affected by that.